nxcomp/Makefile.in

```diff
@@ -58,14 +58,15 @@ LIBS        = @LIBS@
 # Other autoconfigured settings, not used at the moment.
 #
 
-srcdir      = @srcdir@
-prefix      = @prefix@
-exec_prefix = @exec_prefix@
-bindir      = @bindir@
-man1dir     = @mandir@/man1
-VPATH       = @srcdir@
-libdir      = @libdir@
-includedir  = @includedir@
```

Comment:
unnecessary whitespace changes in this commit, please remove them from this commit.